lighthouse-core/scripts/compare-runs.js

```diff
@@ -27,7 +27,7 @@ const {ProgressLogger} = require('./lantern/collect/common.js');
 const LH_ROOT = `${__dirname}/../..`;
 const ROOT_OUTPUT_DIR = `${LH_ROOT}/timings-data`;
 
-const argv = yargs
+const rawArgv = yargs
```

Comment:
@connorjclark I tried to minimize the changes needed in here. I don't think I changed any semantics